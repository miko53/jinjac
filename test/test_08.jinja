new trial with
{{ data_value[0] }} gives the result of an_array at offset 0
{{ data_value[1] }} gives the result of an_array at offset 1
{{ data_value[2] }} gives the result of an_array at offset 2
--
{{ data_value_dbl[0] }} gives the result of an_array at offset 0
{{ data_value_dbl[2] }} gives the result of an_array at offset 2
--
{{ data_value_str[2] }} gives the result of an_array at offset 2
{{ data_value_str[1] }} gives the result of an_array at offset 1
{{ data_value_str[0] }} gives the result of an_array at offset 0


