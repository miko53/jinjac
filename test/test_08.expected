new trial with
10 gives the result of an_array at offset 0
20 gives the result of an_array at offset 1
15 gives the result of an_array at offset 2
--
0.586000 gives the result of an_array at offset 0
159.546000 gives the result of an_array at offset 2
--
AADDEGG gives the result of an_array at offset 2
GTRV gives the result of an_array at offset 1
DES gives the result of an_array at offset 0
--> Now with Identifier: 
15 gives the result of an_array at offset 2
---


